### DISCUSSION TITLE
Documentation - what should I use for an example app for the tutorial?

### DISCUSSION THREAD

**Discussion by joebudi** [Budibase/budibase#627] [Help] [answered]
I am updating the docs and I want to include an example app for the tutorial.

I am considering using a simple CRM or employee contact list as an example, but I happy to welcome other ideas?

**Comment by shogunpurple** [comment, marked as answer]
I like it when tutorials have a "simple" and "advanced" project.

For the simple one, maybe a contact list or todo list would be nice.

For the "advanced" example, a CRM with some nice, simple entities would be a good tutorial piece. This is because:

- It gives a chance to show all the budibase features and build from the previous tutorial
- People know what a CRM is, and if they don't it's quite easy to explain
- It performs actions that are common in business applications. It's quite generic

**Comment by mjashanks** [comment]
I think the main goals are
- Something everyone understand
- Something that can start simple, and progress to cover all the features

A contact list may be too simple
A CRM may not be well understood

How about a task list
- Task has Name, Notes, Deadline, Status, Owner < covers some basic features
- Group task into projects < here, we get into relationships
- Add some automations, based on status / owner change

**Comment by joebudi** [comment]
This is great. As @shogunpurple suggested, I believe a CRM (advanced) and a to-do/task list (simple) might be the answer - CRMs are pretty well known within a business setting.
